src/main/java/org/jenkinsci/remoting/protocol/IOHub.java

```diff
@@ -292,6 +292,10 @@ public final void removeInterestConnect(SelectionKey key) {
      * @param key the key.
      */
     public final void addInterestRead(SelectionKey key) {
+        if (LOGGER.isLoggable(Level.FINEST)) {
```

Comment:
I thought you were going to pull this out to a class level constant initialized on classloading so that hotspot can reduce to no-op